massa-ledger-worker/src/ledger_db.rs

```diff
@@ -398,6 +418,31 @@ impl LedgerDB {
             .create_cf(METADATA_CF, &db_opts)
             .expect("Error creating metadata cf");
     }
+
+    pub fn set_final_state(&mut self, data: &[u8]) -> Result<(), ModelsError> {
+        let handle = self.db.cf_handle(FINAL_STATE_CF).expect(CF_ERROR);
+        let mut batch = WriteBatch::default();
+        batch.put_cf(handle, LEDGER_FINAL_STATE_KEY, data);
+
+        self.db.write(batch).expect(CRUD_ERROR);
+
+        Ok(())
+    }
+
+    pub fn get_final_state(&self) -> Result<Vec<u8>, ModelsError> {
+        let handle = self.db.cf_handle(FINAL_STATE_CF).expect(CF_ERROR);
+        let opt = ReadOptions::default();
+        let mut final_state = Vec::new();
+
+        let db_iterator = self.db.iterator_cf_opt(handle, opt, IteratorMode::Start);
+
+        // Iterates over the whole database
+        for (_key, entry) in db_iterator.flatten() {
+            final_state.extend_from_slice(&entry);
+        }
```

Comment:
If it's close enough to trivial for the db_iterator to get slices instead of vecs (is there an `iterator_pinned_cf`?), then that might be nice here, but don't spend too much time on it.

Reply:
My bad. Didn't realize github interface took me to an older commit.